rules/windows/command_and_control_iexplore_via_com.toml

```
@@ -23,15 +23,18 @@ type = "eql"
 
 query = '''
 sequence by host.id, process.entity_id with maxspan = 1s
-  [process where event.type:"start" and process.parent.name:"iexplore.exe" and process.parent.args:"-Embedding"]
+  [process where event.type : "start" and process.parent.name : "iexplore.exe" and process.parent.args : "-Embedding"]
```

for things that are not dictated by user input but instead by the sensor such as `event.type`, these should stay as case sensitive, since they are consistent

```suggestion
  [process where event.type == "start" and process.parent.name : "iexplore.exe" and process.parent.args : "-Embedding"]
```